### ISSUE TITLE
qt submodule needs updating

### ISSUE THREAD

**Issue by innovara** [wkhtmltopdf/wkhtmltopdf#5277]
I know that according to https://github.com/wkhtmltopdf/wkhtmltopdf/issues/5160 this project is no longer maintained however this is an effortless fix.

Currently the qt submodule is on commit [cb88757](https://github.com/wkhtmltopdf/qt/tree/cb88757e7c9383a0504689c8373c101a44214132) which doesn't include subsequent commits that fixed issues with gcc 9+.

So when you do:

`git clone --recursive https://github.com/wkhtmltopdf/wkhtmltopdf.git`

you get a version that doesn't compile with gcc 9+, until you do:

`git submodule update --recursive --remote`

I discovered this using wkhtmltopdf/packaging to build a package for Fedora 37 which I eventually managed to do for aarch64 and x86_64. I will open a PR with my work in due course.

**Comment by innovara** [comment]
For anyone interested, this is the PR that adds Fedora 36 and 37 build targets: https://github.com/wkhtmltopdf/packaging/pull/135

**Comment by ashkulz** [comment]
@innovara you're right, I ported them to the [0.12.6.x branch](https://github.com/wkhtmltopdf/wkhtmltopdf/tree/0.12.6.x) but not to master :see_no_evil:
